src/ir/auth_logic/universe_relation_insertion.cc

```diff
@@ -35,6 +35,8 @@ std::string TypeToUniverseName(const datalog::ArgumentType& arg_type) {
       return "isPrincipal";
     case datalog::ArgumentType::Kind::kCustom:
       return absl::StrCat("is", arg_type.name());
+    default:
+      return "";
```

Comment:
Instead of `return ""`, shouldn't we do `CHECK(false)`? e.g.,

https://github.com/google-research/raksha/blob/56203028becf8fb196ea641cc8a663c059ee20b7/src/ir/proto_to_ir.cc#L76

Reply:
fixed